packages/cli/src/databases/repositories/execution.repository.ts

```diff
@@ -71,6 +75,14 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 		private readonly executionDataRepository: ExecutionDataRepository,
 	) {
 		super(ExecutionEntity, dataSource.manager);
+
+		if (!inTest) {
+			if (config.getEnv('executions.pruneData')) {
+				setInterval(async () => this.pruneOlderExecutions(), 60 * 60 * 1000); // Every hour
```

Comment:
Go has a lovely practice of bundling amounts of milliseconds into readable constants, e.g. `60 * MINUTE`